low-level/sensors/impl/generic/iem_sensor.py

```diff
@@ -158,33 +158,24 @@ def run(self):
             self._read_iem()
 
         except IOError as io_error:
-            if io_error.errno == errno.ENOENT:
-                logger.debug(f"IEMSensor, self.run, {io_error.args} {io_error.filename}")
-            elif io_error.errno == errno.EACCES:
-                logger.error(f"IEMSensor, self.run, {io_error.args} {io_error.filename}")
-            else:
-                logger.error(f"IEMSensor, self.run, {io_error.args} {io_error.filename}")
-            self._scheduler.enter(10, self._priority, self.run, ())
+            raise Exception(f"IEMSensor, self.run, {io_error.args} {io_error.filename}")
         except Exception as exception:
-            logger.error(f"IEMSensor, self.run, {exception.args}")
-            self._scheduler.enter(10, self._priority, self.run, ())
+            raise Exception(f"IEMSensor, self.run, {exception.args}")
```

Comment:
`raise with e` as described in the previous file.

Reply:
As justified above.